lib/http.go

```diff
@@ -0,0 +1,71 @@
+package lib
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+const jsonMimeType = "application/json"
```

Comment:
Probably nearly always true that we always use application/json, but I think zip output doesn't, there might be one or two other methods.

Reply:
agreed. I think grouping MIME types supported by a package into a set of constants like this is a healthy practice.